x/log_writer.go

```diff
@@ -41,7 +41,9 @@ const (
 	backupTimeFormat = "2006-01-02T15-04-05.000"
 	bufferSize       = 256 * 1024
 	flushInterval    = 10 * time.Second
-	VerificationText = "Hello World"
+	//  old logs before https://github.com/dgraph-io/dgraph/pull/8323 contain deprecated verification text in header
+	VerificationText_Deprecated = "Hello World"
+	VerificationText            = "dlroW olloH"
```

Comment:
is it really supposed to be olloH? or olleH?

Reply:
Good catch :)  all we need is a string of length 11 that is different from "Hello World".  I can change it but it wouldn't make a difference